typescript/common-resolvers/src/import/format-coco/index.ts

```diff
@@ -216,19 +216,26 @@ async function importCocoAnnotationsIntoLabels(
   cocoImageIdToLabelFlowImageId: Map<number, string>,
   cocoCategoryIdToLabelFlowLabelClassId: Map<number, string>,
   { repository, req, user }: Context
-) {
+): Promise<{ skippedCrowdAnnotations: number }> {
   const indexedCocoImages = annotationFile.images.reduce(
     (imagesMap, image) => imagesMap.set(image.id, image),
     new Map<number, CocoImage>()
   );
+  let skippedCrowdAnnotations = 0;
   await Promise.all(
     annotationFile.annotations.map(async (annotation) => {
       if (!cocoImageIdToLabelFlowImageId.has(annotation.image_id)) {
         throw new Error(
           `Image ${annotation.image_id} referenced in annotation does not exist.`
         );
       }
-
+      if (annotation.iscrowd === 1) {
+        skippedCrowdAnnotations += 1;
+        console.log(
+          `Skipped annotation ${annotation.id}: crowd/RLE format no yet supported`
+        );
```

Comment:
Why do we need to display this in the backend console? Why is it not a warning?

Reply:
Just keeping the same logic as in line 251/258 : `console.log(`Created annotation ${annotation.id}`);`
But yeah we probably shouldn't have any of them permanently in the backend logs ...